### PR TITLE
DOC: fill in missing docstrings for Series.is_in

### PR DESCRIPTION
**Pull request by natmokval**
xref #71

### CODE REVIEW

File: narwhals/series.py [hunk 1]
```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this Series are in the other Series.
+
+        Arguments:
+            other: Series or sequence of primitive type.
```
Comment (MarcoGorelli) [inline narwhals/series.py:447]: this should just be "sequence of primitive type"

File: narwhals/series.py [hunk 2]
```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this Series are in the other Series.
```
Comment (MarcoGorelli) [inline narwhals/series.py:444]:
```
Check if elements of this Series are in a given sequence.
```

we don't support `other` being a Series of lists here as Polars does, we can only accept a single sequence

File: narwhals/series.py [hunk 3]
```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
...
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> s_pd = pd.Series([1, 2, 3])
+            >>> s_pl = pl.Series([1, 2, 3])
+
+            We define a library agnostic function:
+
+            >>> def func(s_any):
+            ...     s = nw.from_native(s_any, series_only=True)
+            ...     s = s.is_in(pl.Series([3, 2, 8]))
```
Comment (MarcoGorelli) [inline narwhals/series.py:460]: we shouldn't use anything polars-specific in these examples, can you just use `is_in([3, 2, 8])`?

File: narwhals/series.py (hunk 1 above)
Comment (natmokval) [reply narwhals/series.py]: thanks, I corrected the description of `other`

File: narwhals/series.py (hunk 2 above)
Comment (natmokval) [reply narwhals/series.py]: thanks, I updated summary.

File: narwhals/series.py (hunk 3 above)
Comment (natmokval) [reply narwhals/series.py]: done

File: narwhals/series.py [hunk 4]
```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this sequence are present in the other Series.
+
```
Comment (DeaMariaLeon) [inline narwhals/series.py:445]: We are checking if the Series is in the `other` but `other` is not a Series. 

So I think it should be:
Check if the elements of this Series are in the `other` sequence.

**Comment by DeaMariaLeon** [comment]
Thanks @natmokval, I commented..

File: narwhals/series.py (hunk 4 above)
Comment (natmokval) [reply narwhals/series.py:445]: Thanks @DeaMariaLeon for the comment, I corrected the summary

**Comment by DeaMariaLeon** [comment]
Thank you @natmokval!

**Pull request merged**